devel/libcdio/Portfile

```diff
@@ -33,6 +33,12 @@ checksums           rmd160 6b0e4917cd2dd5cac0b7afdb70789bf0ad2aed79 \
 # doesn't understand the pramga usage in the Tiger headers - gcc bug 50909
 patchfiles          patch-libcdio-lib-iso9660-move-pragma.diff
 
+# Fix the crash reading CD on macos Ventura
+# Remove the unecessary addtional byte added to the TOC buffer
+if {${macos_version_major} >= 13} {
```

Comment:
I thought a `macos_version_major` would be an equivalent check since I imagine it is set only on macos/darwin systems.
And why using `os.major` instead of `macos_version_major`? A matter of convention?
Thank you for your feedback. I'll apply the changes right now.

Reply:
I agree with you that it's technically possible to get the same result with a `macos_version_major` check. But if you `grep` the Port tree you'll see that it is used ionly n one Portfile (but not for checking a version, but in the name of a SDK) and in a single PortGroup. So yes, it's a matter of convention to use `os.platform` and `os.major`.

Reply:
OK thank you for the explanation.